### ISSUE TITLE
Define metadata about bundled software modules & base system, next to main application

### ISSUE THREAD

**Issue by kossebau** [AppImage/AppImageSpec#25]
As a consumer of an AppImage instance I currently miss any information about the additional software bundled with the main application. Same as creator of AppImage instances I miss a specified way to note what is used during the creation. Which can be an issue e.g. when it comes to know if bundled software might have known security issues or when the license of the bundled software requires that its use is made known to the consumer.

So it would be good to have some specified way to notate what ingredients are used, so that this information can be queried for a given AppImage instance.

**Comment by probonopd** [comment]
Good point. I can clearly see why this would be desirable. Not sure it should be part of the AppImage spec, though. AppImage is really meant as a container format that is largely agnostic to what goes inside (like a ISO or zip but self-mounting; comparable to a .dmg on the Mac).

That being said, we could always _recommend_ to put in additional metadata. Before we invent something from scratch, do you think it might be worthwhile to discuss this idea with the AppStream people?

**Comment by kossebau** [comment]
I have not yet thought more about it, just found this something lacking when thinking about how to do proper AppImages,
For docker I had come across http://label-schema.org/ as one approach, just listing here as reference.
No time to work on/think about this myself currently.
